test/k8sT/Policies.go

```diff
@@ -406,12 +411,99 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 				return
 			}
 
-			By("Installing egress default-deny")
+			By("Installing knp egress default-deny")
 
 			eps := kubectl.CiliumEndpointPolicyVersion(ciliumPod)
 
 			_, err := kubectl.CiliumPolicyAction(
-				helpers.KubeSystemNamespace, denyEgress, helpers.KubectlApply, 300)
+				helpers.KubeSystemNamespace, knpDenyEgress, helpers.KubectlApply, 300)
+			Expect(err).Should(BeNil(),
+				"L3 deny-egress Policy cannot be applied in %q namespace", helpers.DefaultNamespace)
+
+			err = helpers.WaitUntilEndpointUpdates(ciliumPod, eps, 4, kubectl)
```

Comment:
This `WaitUntilEndpointUpdates` can be deleted. The CiliumPolicyAction  takes care of that.

Reply:
done